cms/static/sass/views/_settings.scss

```diff
@@ -438,6 +438,24 @@
           padding-bottom: 0;
         }
 
+        // course details that should appear more like content than elements to change
+        .is-not-editable {
```

Comment:
Has a UX person signed off on this? I'm not sure if this follows our class name pattern.

I'm also not clear why the styling is not on the input itself (based on the readonly property on the input).